app/javascript/mastodon/containers/mastodon.js

```diff
@@ -78,13 +78,15 @@ export default class Mastodon extends React.PureComponent {
       <IntlProvider locale={locale} messages={messages}>
         <ReduxProvider store={store}>
           <ErrorBoundary>
-            <BrowserRouter basename='/web'>
+            <BrowserRouter>
               <ScrollContext shouldUpdateScroll={this.shouldUpdateScroll}>
                 <Route path='/' component={UI} />
               </ScrollContext>
             </BrowserRouter>
 
-            <Helmet defaultTitle={title} titleTemplate={`%s - ${title}`} />
+            <Helmet defaultTitle={title} titleTemplate={`%s - ${title}`}>
+              <meta name='robots' content='noindex' />
+            </Helmet>
```

Comment:
Hm, I thought of it as the default that would get overwritten by components that allow indexing. I guess it depends on if there's an intermediary state where this gets rendered on a route that would allow indexing otherwise, and whether Google's rendering engine waits for DOM to stop updating or not.

Reply:
Right, though I believe the `Status` and `AccountTimeline` components are loaded asynchronously, so they probably will cause `noindex` to be added and committed to the DOM in most cases, which I believe would stop Google from indexing anything.